tools/osbuild-mpp

```diff
@@ -642,6 +649,11 @@ LocalFileSigLevel = Optional
                 pkg["name"],
                 pkg["version"],
                 pkgdata["Architecture"],
+                pkgdata.get("Licenses"),
+                pkgdata.get("Description"),
+                pkgdata.get("sourcerpm"),
+                pkgdata.get("Build Date "),
```

Comment:
Last nitpick from me: here it is called `Build Date`, elsewhere it is called `buildtime`. Judging by the name I would say that `build date` shall have only date inside, where `build time` is more about the clock time. What is true here and if we should unify this?

Reply:
Build Date is the string that pacman produces in its output, so it is what we need to look for. I'm not sure how we can do anything about the name?

I also don't think it is useful to try to convert these to some shared format between rpm and pacman date/time formats. You will be using either, and expect time/date stings in the format of the system you're using. Both formats contain both the date and the time in the string though.

Reply:
You added a new field to `PkgInfo` called `buildtime`, why not call that `builddate` to match the field being fetched?

Reply:
There is a new field in the PkgInfo object. When using the rpm backend its content is taken from the libdnf object as "pkg.buildtime", and when using the pacman backend it is taken from the pacman output where the output contains "Build Date". Neither of these are under the control of osbuild-mpp, they are what they are. The only thing we can chose is the PkgInfo field name to store this in. 

For that, I think buildtime fits better. Both what it is (it contains the time) and from how it is more likely to be used (most users of osbuild use rpm, and not many use pacman).

Reply:
Sounds like a reasonable explanation to me.